deploy/olm-catalog/knative-kafka-operator/0.6.0/knative-kafka-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -156,9 +156,9 @@ spec:
         serviceAccountName: knative-kafka-operator
     strategy: deployment
   installModes:
-  - supported: true
+  - supported: false
     type: OwnNamespace
-  - supported: true
+  - supported: false
```

Comment:
this means, you need to update the PR 420 on the community-operator repo :-)

Reply:
Absolutely, as soon as this PR merges